Use AC_ARG_WITH correctly so that the advertised configure options
actually exist under their expected names.

https://github.com/frankheckenbach/ftgl/pull/13
--- m4/gl.m4.orig	2019-02-07 15:39:58.000000000 -0600
+++ m4/gl.m4	2021-02-10 21:21:30.000000000 -0600
@@ -7,11 +7,11 @@
 AC_REQUIRE([AC_PATH_X])
 AC_REQUIRE([AC_PATH_XTRA])
 
-AC_ARG_WITH([--with-gl-inc],
+AC_ARG_WITH([gl-inc],
     AC_HELP_STRING([--with-gl-inc=DIR],[Directory where GL/gl.h is installed]))
-AC_ARG_WITH([--with-gl-lib],
+AC_ARG_WITH([gl-lib],
     AC_HELP_STRING([--with-gl-lib=DIR],[Directory where OpenGL libraries are installed]))
-AC_ARG_WITH([--with-glu-lib],
+AC_ARG_WITH([glu-lib],
     AC_HELP_STRING([--with-glu-lib=DIR],[Directory where OpenGL GLU library is installed]))
 
 AC_LANG_SAVE
--- m4/glut.m4.orig	2019-02-07 15:39:58.000000000 -0600
+++ m4/glut.m4	2021-02-10 21:21:21.000000000 -0600
@@ -8,9 +8,9 @@
 AC_REQUIRE([AC_PATH_XTRA])dnl
 AC_REQUIRE([FTGL_CHECK_GL])dnl
 
-AC_ARG_WITH([--with-glut-inc],
+AC_ARG_WITH([glut-inc],
     AC_HELP_STRING([--with-glut-inc=DIR],[Directory where GL/glut.h is installed (optional)]))
-AC_ARG_WITH([--with-glut-lib],
+AC_ARG_WITH([glut-lib],
     AC_HELP_STRING([--with-glut-lib=DIR],[Directory where GLUT libraries are installed (optional)]))
 
 AC_LANG_SAVE
